routemaster/client.rb

```diff
@@ -30,36 +30,49 @@ def configure
         end
       end
 
-      def created(topic, callback, timestamp = nil)
-        _send_event('create', topic, callback, timestamp)
+      def created(topic, callback, timestamp = nil, t: nil)
+        _warn_timestamp_deprecation(timestamp)
+        _send_event('create', topic, callback, t: t || timestamp)
       end
 
-      def created_async(topic, callback, timestamp = nil)
-        _send_event('create', topic, callback, timestamp, async: true)
+      def created_async(topic, callback, timestamp = nil, t: nil)
+        _warn_timestamp_deprecation(timestamp)
+        _warn_async_deprecation
+        _send_event('create', topic, callback, t: t || timestamp, async: true)
       end
 
-      def updated(topic, callback, timestamp = nil)
-        _send_event('update', topic, callback, timestamp)
+      def updated(topic, callback, timestamp = nil, t: nil)
```

Comment:
the readme says
```
Routemaster::Client.updated('widgets', 'https://app.example.com/widgets/2', async: true)
```

I might be missing something but how do we pass the `async` flag here to the method (and to `_send_event`?) (Same for the other actions)